ports/geckolib/glue.rs

```diff
@@ -338,6 +374,54 @@ pub extern "C" fn Servo_StyleSet_Drop(data: RawServoStyleSetOwned) -> () {
     let _ = data.into_box::<PerDocumentStyleData>();
 }
 
+
+#[no_mangle]
+pub extern "C" fn Servo_ParseProperty(property_bytes: *const u8,
```

Comment:
That's just one of a bunch of things are now broken but used to work. I wish bors-servo had got around to merging it a few days ago when it applied cleanly. I'll look into it on Tuesday.